packages/ui/src/components/ProjectCard/ProjectCard.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { ProjectCard } from './ProjectCard';
+
+type StatusArray = {
+  status: 'idle' | 'doing' | 'review' | 'done';
+  text: string;
+}[];
+
+const title = 'Zaawansowany React i Node';
+const date = new Date();
+const statusArray: StatusArray = [
+  { status: 'idle', text: 'Nie rozpoczęty' },
+  { status: 'doing', text: 'W trakcie' },
+  { status: 'review', text: 'W ocenie' },
+  { status: 'done', text: 'Zakończony' },
+];
+
+describe('ProjectCard', () => {
+  statusArray.map((el) => {
+    return it(`if status is equal to${el.status} should display ${el.text}`, () => {
+      if (el.status === 'done') {
+        render(<ProjectCard status="done" image="test" title={title} url="test" points={100} pointsMax={200} />);
+      } else {
+        render(<ProjectCard status={el.status} image="test" title={title} url="test" date={date} />);
+      }
+
+      expect(screen.getByText(el.text)).toBeInTheDocument();
+    });
+  });
```

Comment:
🙃 Add a new line between different test cases. It will make the code more consistent.